README.md

```diff
@@ -217,6 +218,34 @@ are not part of the standard Elastic stack, but can be used to enrich it with ex
 The documentation for these extensions is provided inside each individual subdirectory, on a per-extension basis. Some
 of them require manual changes to the default ELK configuration.
 
+### How can I use Logspout?
+
+Logspout is a log router for Docker containers that runs inside Docker. It attaches to all containers on a host, then routes their logs wherever you want. It also has an extensible module system.
+
+Add below configuration to your docker-stack.yml to enable logspout. Please note that we will use Logspout on all the servers to pass the logs to Logstash. 
```

Comment:
nit: Change the wording so that you don't use `we`. READMEs should be targeted at reader.